components/rating/rating.tsx

```diff
@@ -0,0 +1,124 @@
+import { Icon, Props as IconProps } from '@geist-ui/react-icons'
+import React, { useEffect, useMemo, useState } from 'react'
+
+import { GeistUIThemes } from '../themes/presets'
+import { NormalTypes } from '../utils/prop-types'
+import Star from '@geist-ui/react-icons/star'
+import useTheme from '../use-theme'
+import withDefaults from '../utils/with-defaults'
+
+interface Props {
+  type?: NormalTypes
```

Comment:
It would be nice if you could add `size`.

Reply:
Size can be added without any problem 👍

Reply:
Oh I forgot about this 😃 , I added the global size in #531 , you shouldn't need to add the `size` manually, forget this comment, I'll deal with this later.